pipeline/action.py

```diff
@@ -0,0 +1,40 @@
+'''
+Schema of session information.
+'''
+import datajoint as dj
+from pipeline import reference, subject
+
+schema = dj.schema('ttngu207_action',locals())
+
+@schema
+class Weighing(dj.Manual):
+    definition =  """
+    -> subject.Subject
+    ---
+    weight_before: float   # in grams
+    weight_after: float    # in grams
+    """
+
+@schema
+class WhiskerConfig(dj.Manual):
+    definition = """
+    -> subject.Subject
+    ---
+    -> reference.Whisker
+    """
```

Comment:
Now I remember why I named `Whisker` in the reference. I think I just used `WhiskerConfig` to describe the whisker status of a subject, while `Whisker` is a general status of the whiskers. We can also name it `reference.WhiskerConfig`, and here `SubjectWhiskerConfig`.